src/lte.js

```diff
@@ -0,0 +1,13 @@
+const lte = ( input1, input2 ) => {
+  if( input1 <= input2 ) {
```

Comment:
First, indentation.  Second, when you return from within an if statement the result of evaluating the expression, you can just return the result itself:
```es6
const lte = (input1, input2) => input1 <= input2
```